test/io/readfile/read_entire_file.chpl

```diff
@@ -0,0 +1,14 @@
+use IO;
+use Time;
+var f=open("/home/sayan/chapel-1.25.0/examples/f2.txt",
```

Comment:
This should just say `f2.txt` or whatever local path you have. The path present here now will not be present on systems other than your own.